src/haz3lcore/dynamics/Elaborator.re

```diff
@@ -271,7 +272,11 @@ let rec dhexp_of_uexp =
         | InHole(
             InexhaustiveMatch(Some(Common(Inconsistent(Internal(_))))),
           ) =>
+          // TODO: review
```

Comment:
Not problematic for now since currently `DHExp.InexhaustiveCase` behaves exactly the same as `DHExp.InconsistentBranches`